web/sidenav.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:html';
+
+/// Initialize the sidenav event handlers.
+void init() {
+  final document = window.document;
+
+  var leftNavToggle = document.getElementById('sidenav-left-toggle');
```

Comment:
Is this functionality new? I can't find other references to this class.

Reply:
It was in and around here: https://github.com/dart-lang/dartdoc/blob/master/lib/resources/script.js#L2 - just made a bit more terse.